wandb/cli/cli.py

```diff
@@ -423,7 +435,14 @@ def init(ctx, project, entity, reset, mode):
 
 
 @cli.command(
-    context_settings=CONTEXT, help="Upload an offline training directory to W&B"
+    context_settings=CONTEXT,
+    help="""Synchronize offline W&B run data to the cloud.
```

Comment:
For this one, I'm working on a replacement which you can find in `wandb/cli/beta.py`. The description I went with is:

```
  Upload .wandb files specified by PATHS.

  PATHS can include .wandb files, run directories containing .wandb files, and
  "wandb" directories containing run directories.

  For example, to sync all runs in a directory:

      wandb beta sync ./wandb

  To sync a specific run:

      wandb beta sync ./wandb/run-20250813_124246-n67z9ude

  Or equivalently:

      wandb beta sync ./wandb/run-20250813_124246-n67z9ude/run-n67z9ude.wandb
```

I think the last two examples also work for the legacy implementation, but not the first one.